src/background.js

```diff
@@ -111,11 +111,26 @@ async function start() {
     return {
       requestHeaders,
     };
-  }, { urls: [`${SERP_BASE_URL}/search*`]}, ["blocking", "requestHeaders"]);
+  }, { urls: [`${SERP_BASE_URL}/search*`, USE_STAGING ? 'https://staging.ghosterysearch.com/search*' : 'https://ghosterysearch.com/search*']}, ["blocking", "requestHeaders"]);
```

Comment:
Should this be:
```suggestion
  }, { urls: [`${SERP_BASE_URL}/search*`, USE_STAGING ? 'https://staging.glowstery.com/search*' : 'https://glowstery.com/search*']}, ["blocking", "requestHeaders"]);
```

Reply:
No, `*.ghosterysearch.com` is from now considered deprecated and will be removed in next release. This is just for transition period.